_data/committers.yml

```diff
@@ -15,203 +15,206 @@
 #
 # Database of Apache Arrow committers and PMC
 #
-- name: Jacques Nadeau
+
+- name: Wes McKinney
   role: VP
-  alias: jacques
-  affiliation: Dremio
-- name: Ted Dunning
+  alias: wesm
+  affiliation: Ursa Labs / RStudio
+
+- name: Abdelhakim Deneche
   role: PMC
-  alias: tdunning
-  affiliation: MapR
-- name: P. Taylor Goetz
+  alias: adeneche
+  affiliation: Salesforce
+- name: Alex Levenson
   role: PMC
-  alias: ptgoetz
-  affiliation: Monetate
-- name: Julian Hyde
+  alias: alexlevenson
+  affiliation: Twitter
+- name: Andy Grove
   role: PMC
-  alias: jhyde
-  affiliation: Looker
-- name: Reynold Xin
+  alias: agrove
+  affiliation: NVIDIA
+- name: Antoine Pitrou
   role: PMC
-  alias: rxin
-  affiliation: Databricks
-- name: James Taylor
+  alias: apitrou
+  affiliation: Independent / Ursa Labs
+- name: François Saint-Jacques
   role: PMC
-  alias: jamestaylor
-  affiliation: Salesforce
-- name: Julien Le Dem
+  alias: fsaintjacques
+  affiliation: Ursa Labs / RStudio
```

Comment:
@fsaintjacques what's your company called again?

Reply:
:wave:, it's called Optable.